api/src/paths/project/{projectId}/funding-sources/add.ts

```diff
@@ -58,7 +58,7 @@ function addFundingSource(): RequestHandler {
 
       await connection.commit();
 
-      return res.status(200).json(result.id);
+      return res.status(200).json({ id: result.id });
```

Comment:
does changing the return value here not affect anything on the frontend? it might not but just wanted to confirm youve checked by adding a funding source :)

Reply:
yeah, I tested that and it worked.  Let me confirm again ... I'll let you know

Reply:
yes, all good!